Uplift/Views/GymDetail/GymDetailHoursCell.swift

```diff
@@ -207,7 +190,7 @@ class GymDetailHoursCell: UICollectionViewCell {
 extension GymDetailHoursCell: UITableViewDelegate, UITableViewDataSource {
 
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        let numRows = GymDetailHoursCell.hoursData.isDropped ? 6 : 0
+        let numRows = hoursDataIsDropped ? 6 : 0
```

Comment:
Where are these values coming from? Can we store them in constants or a static enum somewhere?

Reply:
its coming from `GymDetail` model. we don't want it to be `static` because thats what was causing the bug in the first place